src/java.base/share/classes/java/lang/module/ModuleDescriptor.java

```diff
@@ -152,28 +158,31 @@ public enum Modifier {
              * module</i> to have an implicitly declared dependence on the module
              * named by the {@code Requires}.
              */
-            TRANSITIVE,
+            TRANSITIVE(AccessFlag.TRANSITIVE.mask()),
 
             /**
              * The dependence is mandatory in the static phase, during compilation,
              * but is optional in the dynamic phase, during execution.
              */
-            STATIC,
+            STATIC(AccessFlag.STATIC.mask()),
```

Comment:
This is actually `AccessFlag.STATIC_PHASE` (`0x0040`), and not `AccessFlag.STATIC` (`0x0020`):
```suggestion
            STATIC(AccessFlag.STATIC_PHASE.mask()),
```

Reply:
> In the current hodgepodge AccessFlag, we have STATIC and STATIC_PHASE, and the incorrect ModuleDescriptor.accessFlags().contains(AccessFlag.STATIC) call is much more subtle, especially to new users of this class. Arguably, this misuse would be way worse than that in the distinct enum case.

Oops, didn't know this already happened. Good spot right there.

Reply:
Corrected to STATIC_PHASE in subsequent push; thanks.